client/src/entrypoints/core.js

```diff
@@ -369,220 +369,217 @@ $(() => {
 // Inline dropdown module
 // =============================================================================
 
-// eslint-disable-next-line func-names
-window.wagtail = (function (document, window, initialWagtail) {
-  const wagtail = initialWagtail || {};
-  if (!wagtail.ui) {
-    wagtail.ui = {};
-  }
-
-  // Constants
-  const DROPDOWN_SELECTOR = '[data-dropdown]';
-  const LISTING_TITLE_SELECTOR = '[data-listing-page-title]';
-  const LISTING_ACTIVE_CLASS = 'listing__item--active';
-  const ICON_DOWN = 'icon-arrow-down';
-  const ICON_UP = 'icon-arrow-up';
-  const IS_OPEN = 'is-open';
-  const clickEvent = 'click';
-  const TOGGLE_SELECTOR = '[data-dropdown-toggle]';
-  const ARIA = 'aria-hidden';
-  const keys = {
-    ESC: 27,
-    ENTER: 13,
-    SPACE: 32
-  };
-
-
-  /**
-   * Singleton controller and registry for DropDown components.
-   *
-   * Mostly used to maintain open/closed state of components and easily
-   * toggle them when the focus changes.
-   */
-  const DropDownController = {
-    dropDowns: [],
-
-    closeAllExcept(dropDown) {
-      const index = this.dropDowns.indexOf(dropDown);
-
-      this.dropDowns.forEach((item, i) => {
-        if (i !== index && item.state.isOpen) {
-          item.closeDropDown();
-        }
-      });
-    },
-
-    add(dropDown) {
-      this.dropDowns.push(dropDown);
-    },
-
-    get() {
-      return this.dropDowns;
-    },
+const wagtail = window.wagtail || {};
```

Comment:
Not really sure about this change - by removing the function wrapper, don't we end up dumping a load of constants into the global scope (including some with pretty generic names like `keys`)? Is there a chance that this could clash with other code elsewhere using those variables as non-constants?

(Happy with everything else in the PR though!)

Reply:
It just moves it up to module scope, webpack encapsulates the modules so this shouldn't affect any other files (hence why we need to explicitly add stuff to the ``window`` object now when we want to create globals).